apps/prairielearn/elements/pl-file-upload/pl-file-upload.js

```diff
@@ -11,20 +20,57 @@
 
   class PLFileUpload {
     constructor(uuid, options) {
+      // Not configurable at this point as this matches the request size limit enforced by the server
+      this.maxFileSizeMB = 5;
```

Comment:
This doesn't actually match the server-enforced limit, which is 7.5MiB. This is intentional to account for the overhead of base64-encoding files. I'd like to make that explicit in the comment here though.